Server/Repositories/Interfaces/IUsersRepository.cs

```diff
@@ -16,5 +17,10 @@ public interface IUsersRepository
         public Task<bool> SetNewPasswordForUser(User user, string rawPassword);
         public Task<User> GetUserByRefreshTokenAsync(string refreshToken);
         public Task<User> GetUserByEmailAsync(string email);
+        /// <summary>
+        ///     Funkcja zwracająca top 3 użytkowników
+        /// </summary>
+        /// <returns></returns>
+        public Task<List<Top3UserResponseDTO>> GetTop3Users();
```

Comment:
Because it is an interface in my opinion the return type should be IEnumerable<> rather then  List<>